client/src/quic_client.rs

```diff
@@ -78,8 +78,13 @@ impl TpuConnection for QuicTpuConnection {
             .map(|tx| bincode::serialize(&tx).expect("serialize Transaction in send_batch"))
             .collect::<Vec<_>>();
 
+        let buffer_refs = buffers.iter().map(|buf| &buf[..]).collect::<Vec<_>>();
```

Comment:
I think if you leave the signature of send_batch unchanged, we do not even need this conversion to get the references:
pub async fn send_batch(&self, buffers: &[Vec<u8>])

Reply:
Also, with the definition of this function in the trait -- do we need to repeat here?